modules/core/src/main/java/org/apache/ignite/internal/util/portscanner/GridJmxPortFinder.java

```diff
@@ -25,12 +25,14 @@
 import java.net.UnknownHostException;
 import java.nio.channels.FileLock;
 import org.apache.ignite.IgniteSystemProperties;
+import org.apache.ignite.SystemProperty;
 
 /**
  * Ignite port scanner.
  */
 public class GridJmxPortFinder {
     /** Environment variable for overriding JMX port. */
+    @SystemProperty(value = "JMX port.", type = Integer.class)
```

Comment:
"Sets a JMX port for the Ignite instance. Defaults to X."

Reply:
I think we describe the value and not action. So "sets" is wrong word - this property is just a jmx port value.
Makes sense?